__tests__/invalid-data/invalid2.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Recommend the filename includes the reason why the test case is invalid. E.g., "invalid-kind-enum.json". Without this context we must hunt for the reason.

Reply:
changed the name to invalid-kind-enum.json for better readability